_events/2015/2015-02-19-meetup.md

```diff
@@ -0,0 +1,8 @@
+---
+location: Christchurch Office, Catalyst IT Limited, Level 1, 284 Kilmore St, Christchurch
+time: 7:00pm
+title: Monthly Meetup
+
+---
+
+TBA. We'll update these details closer to the time. :)
```

Comment:
```suggestion
```

we did not

Reply:
Good call. I missed those on the first pass. Removed now along with the others.